de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc

```diff
@@ -101,6 +101,66 @@ Now press the link btn:[Workbench].
 
 Done. You installed and successfully started Virtual Satellite Developer IDE.
 
+==== IDE Git setup
+
+Once you have installed your Virtual Satellite Developer IDE, you need to set up your Git IDE. 
+Following the next steps will help you with the progress.
+
+
+[%header,cols=">8,62a,40a"] 
+|===
+
+|Step
+|Task
+|Image
+
+|{counter:steps}.
+|Start with opening a new perspective by clicking the "window" button in your menu. +
+Select the option "Git". 
+|image::chapterIdeInstallation/subsectionGitIdeSetup/2_OpenPerspective.png[Open perspective]
+
+|{counter:steps}.
+|In the next step you have to select a repository source.
+Choose "clone URl" and continue. +
+After that you need to visit www.github.com and choose the repository you want to clone. +
+Click the green button in the upper right corner which says "clone or download" and copy the link.
+|image::chapterIdeInstallation/subsectionGitIdeSetup/4.1_CloneGitUrl.PNG[Clone Git URL]
+
+|{counter:steps}.
+|Enter the link of the source repository into the free field "URl" and authenticate yourself in the section beneath. +
+In the next step you can select branches to clone from the remote repository. In this case, just let them all selected and continue. +
+Now you have to choose a directory as a local destination for VirtualSatellite4-Core to finish the process. 
+|image::chapterIdeInstallation/subsectionGitIdeSetup/5_InsertGitUrl.PNG[Insert Git URL]
+
+|{counter:steps}.
+|After that, you select the branch "development" in the upper left corner of your window and double click to check out the branch. 
+|image::chapterIdeInstallation/subsectionGitIdeSetup/8_CheckoutBranch.PNG[Check out development branch]
+
+|{counter:steps}.
+|Now you have to open the folder "Working Tree" in your Git repositories on the left side and scroll down until you find the folder "de.dlr.sc.virsat.target". +
+Open it and then select "virsat.target". +
+You need to wait until the progress is fully loaded. +
+Then set it as your active target platform. The button is in the upper right corner.
+|image::chapterIdeInstallation/subsectionGitIdeSetup/10_OpenVirsatTarget.PNG[Open Virsat Target]
+
+|{counter:steps}.
+|After that you need to switch from the Git perspective to the Plug-in development perspective in the upper right corner. +

```

Comment:
Instead of the series of steps described here I usually just import the projects form the git perspective via:
Working Tree -> Import Projects -> Finish
Any reason we need a more complicated workflow with perspective switching?

Reply:
Yes, that seems to be much easier. Changed that.